### PR TITLE
Support generating OWNERS for single repos

### PR DESCRIPTION
**Pull request by 0xmichalis**

### CODE REVIEW

File: tools/populate-owners/main.go [hunk 1]
```diff
@@ -457,8 +467,28 @@ func pullOwners(directory string) (err error) {
 	return nil
 }
 
+var (
+	org  = flag.String("org", "", "Organization to use for updating OWNERS")
```
Comment (stevekuznetsov) [inline tools/populate-owners/main.go]: Could default here to `*`

Comment (kargakis) [reply tools/populate-owners/main.go]: done

**Comment by kargakis** [comment]
> Is this safe? Can you create alias collisions with this?

Yeah, this is probably possible. I can avoid updating aliases when a single repo is provided. If you need to update your aliases you will have to run with the global options. How does that sound?

**Comment by stevekuznetsov** [comment]
Is there a better way to make this faster? Can we fetch in parallel? That would solve the issue but not cause a weird execution flow

**Comment by kargakis** [comment]
The problem with running this globally is that usually you get diff for which you have no control over (no OWNER of the most updated repos) so you end up commiting only the changes for the repo(s) you care about because it will make the process of merging faster.

**Comment by stevekuznetsov** [comment]
I don't think anyone should run the tool and commit only partial results. If you run the tool you should update all repos.

**Comment by petr-muller** [comment]
I know I'm coming late but to me the approach in https://github.com/openshift/release/pull/2373#issuecomment-447062229 would be totally fine. We ask people to use this tool more with our enforcing presubmit (for newly configured components) and it makes total sense to just commit their OWNERS file in that scenario - I can totally understand people just do not want to care about other components. I actually have this in my wannado list too somewhere, too.

**Pull request closed without merging**